mocha.opts

```diff
@@ -3,4 +3,5 @@
 --require src/test/helpers/mocha-bootstrap.js
 --recursive
 --timeout=1000
+--exit
```

Comment:
I go back and forth on including this flag. I'd stick to not including it so that you can still see _all_ the errors that occur during a test run (which is a general philosophy for testing in general). What do you think?

Reply:
I think you're confusing `--exit` and `--bail`?  The latter exits after the first failure.  This one just makes Mocha not hang forever at the end, which happened to me at least once.

Reply:
You're right, so I went and looked at the `--exit` [documentation](https://mochajs.org/#-exit) just now.

It looks like having to add `--exit` is indicative of an underlying problem of a test not cleaning up after itself. If that issue doesn't exist, we shouldn't add `--exit` as it could hide potential issues

Reply:
If nothing, I'd say maybe we still have a dangling `setInterval` in WorkQueue? Can we try with a `afterEach` to `.stop()` all the queues and see if that removes the need for `--exit`?

Reply:
Fixed, it was a failure to call `stop()` on the last test (since stop was in beforeEach not afterEach.  Thanks!)